ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-v4-evpn.md

```diff
@@ -189,17 +189,23 @@ interface Management1
 
 ### Router BGP EVPN Address Family
 
-#### Router BGP EVPN MAC-VRFs
+#### EVPN Peer Groups
 
-##### VLAN aware bundles
+| Peer Group | Activate |
+| ---------- | -------- |
+| EVPN-OVERLAY-PEERS | True |
+| IPv4-UNDERLAY-PEERS | False |
+| MLAG-IPv4-UNDERLAY-PEER | False |
+
+### Router BGP MAC VRF for multiple VLANs (Vlan Aware Bundles) Instances
```

Comment:
IMO the old title was more clear. I suggest `Router BGP VLAN Aware Bundles`, `Router BGP VRFs` and `Router BGP VLANs`

Reply:
I agree, updated accordingly